hswindow/hseditor/hsdc6editor/dc6_editor.go

```diff
@@ -110,14 +109,6 @@ func (e *DC6Editor) UpdateMainMenuLayout(l *g.Layout) {
 	*l = append(*l, m)
 }
 
-// RegisterKeyboardShortcuts adds a local shortcuts for this editor
-func (e *DC6Editor) RegisterKeyboardShortcuts(inputManager *hsinput.InputManager) {
-	// Ctrl+Shift+S saves file
-	inputManager.RegisterShortcut(func() {
-		e.Save()
-	}, g.KeyS, g.ModShift+g.ModControl, false)
-}
```

Comment:
is this `ctrl`+`shift`+`s` hotkey being registered elsewhere, now?

Reply:
it isn't, however, I'm not really sure if this shortcut is necessary. saving editor data takes place on ctrl+s and this shortcut is registered in hsapp.